### PR TITLE
Issue 4 - Implement Logging System

### PR DESCRIPTION
**Pull request by takmo**
Not quite ready for merge yet, but good to have this here for Issue 4.

Needs a bit of changing still, but looks like good work to me. See comments below.

### CODE REVIEW

File: server/logger.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import time
+import logging
+
+def Logger(name):
```
Comment (takmo) [inline server/logger.py]: This function should create and attach the appropriate handlers and formatters to the _root_ log. Moreover, it should verify that this only gets done once, as we don't want many identical handlers attached to the same log. Then return a logger for `name`.

The goal is to have every different subsystem writing to a different sub-log, but having them all aggregated and written by the root log.

Comment (takmo) [inline server/logger.py]: Also, perhaps rename it to get_logger? If only to match the naming scheme of other functions?

**Comment by Celdir** [comment]
New code has arrived. :D

**Comment by Celdir** [comment]
Looks content.

**Comment by takmo** [comment]
Fixes #4

**Pull request merged**